plugins/woocommerce/client/admin/package.json

```diff
@@ -302,7 +303,7 @@
 			"env": {
 				"DISABLE_TYPESCRIPT_CHECKING": {
```

Comment:
I think we can remove this functionality now, I only put it in place for ease of developing on the react18 and ts fixes

Reply:
Ok, I've removed it! 🙌  bb5f835f1622503f2c0e237fd2375b2430e3b511